### DISCUSSION TITLE
Duda REPORTE-PLANETAS en test cases públicos

### DISCUSSION THREAD

**Discussion by hansjkl** [IIC2133-PUC/2024-1#25] [Tarea 0] [answered]
Hola, en los test cases publicados 2-3 y 2-3-inc REPORTE-PLANETAS tiene el siguiente formato:

> PLANETAS-ORDENADOS
    PLANETA 5: 2 pedidos
    PLANETA 0: 1 pedidos
    PLANETA 1: 1 pedidos
    PLANETA 4: 1 pedidos
    PLANETA 2: 0 pedidos
    PLANETA 3: 0 pedidos
    PLANETA 6: 0 pedidos
    TOTAL-PEDIDOS-ENTREGADOS: 5

Sin embargo, en el enunciado el número final se presenta con un salto de línea y 4 espacios. 
![image](https://github.com/IIC2133-PUC/2024-1/assets/42097289/47650073-2155-4cf8-ad03-e85b55a3a09b)

¿Cuál de estos formatos debemos considerar para la entrega de la tarea?

**Comment by benjavicente** [comment, marked as answer]
Hola!

Es un problema en el formato de esa tabla en LaTeX, la línea 7 hace wrap, mostrándose visualmente abajo (por eso no hay línea 8).

Acabo de subir el enunciado con el formato arreglado.

**Reply by hansjkl** [reply]
Muchas gracias!
